app/helpers/job_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module JobHelper
+  PERF_CHECK_USER_TYPES = ['admin', 'super', 'user', 'standard', 'read']
```

Comment:
Just FYI these aren't actually codified in the `perf_check` gem itself, they can be defined in `config/perf_check.rb` 

I realize this constant was already here, but we probably have 2 options for the long run:

1) Read these user types from config
2) Codify them in perf check and modify any installs that use them.

Reply:
Alright, I created [an issue](https://github.com/rubytune/perf_check_ci/issues/26) about this.